src/altinn-app-frontend/src/features/form/components/RequiredIndicator.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { getLanguageFromKey } from 'altinn-shared/utils/language';
+import type { ILabelIndicatorProps } from './OptionalIndicator';
+
+export const RequiredIndicator = ({ language }: ILabelIndicatorProps) => {
```

Comment:
Any specific reason to why you are passing the language as a prop, instead of fetching it from redux store?

I think in general it would be good to fetch things that are in the store at the lowest level possible, to avoid prop drilling these all the way from the top.

Reply:
I guess just to keep the component as simple as possible. But seeing as it is a very specific component, I agree that we might as well fetch language directly from store.